src/ray/object_manager/object_directory.h

```diff
@@ -40,6 +40,11 @@ struct RemoteConnectionInfo {
   uint16_t port;
 };
 
+/// Callback for object location notifications.
```

Comment:
I'm fine with moving this to `common.h` too, but it needs a larger scope than just the ObjectDirectory.